lib/game.rb

```diff
@@ -0,0 +1,138 @@
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [[0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8], [0,4,8], [2,4,6]]
+
+  def initialize(player_1 = Player::Human.new("X"), player_2 = Player::Human.new("O"), board = Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  def current_player
+    return @board.turn_count.even? ? @player_1 : @player_2
+  end
+
+  def over?
+    draw? || won? ? true : false
```

Comment:
A ternary operator where both results are literal boolean values is redundant; just return the value of the boolean expression.
```
def over?
  draw? || won?
end
```